gamedb.yaml

```diff
@@ -5069,6 +5069,15 @@
   compat_tool: proton-stable
   launch_options: -force -dx11
 
+- name: "Psychonauts 2"
+  platform: gog
+  id: 1508046796
+  status: playable
+  banner: steam:607080
+  poster: https://cdn2.steamgriddb.com/thumb/7d5b05b45008485350016d586a6a3e2e.jpg
```

Comment:
Don't need poster, `banner: steam:607080` is a shorthand which grabs all the artwork from Steam.

Reply:
Roger that. Is this documented anywhere? I can create a README file for this repo, but I am afraid I don't know enough about the inner workings of the repo to write something exhaustive.

Reply:
Yeah, it is not documented. It is on my TODO list.